update_promotable_jobs.py

```diff
@@ -9,16 +11,26 @@
 
 
 def run():
-    job_data = parse_job_data()
-    jobs = determine_jobs(job_data)
-    for job in jobs:
-        cluster = job_data[job].get('cluster', '')
-        for environment in ['staging', 'production']:
-            update_job_promotion(job, environment, cluster)
-
-def parse_job_data():
+    job_data = parse_job_data(['jobs/app-jobs.yml','jobs/java-docker-jobs.yml'])
+    try:
+       jobs = determine_jobs(job_data)
+       xml_config_file = "jobs/promotion-config.xml"
+       for job in jobs:
+           cluster = job_data[job].get('cluster', '')
+           for environment in ['staging', 'production']:
+               update_job_promotion(job, environment, cluster, xml_config_file)
+    finally:
```

Comment:
It won't offend me at all if you restructure this (or anything else).  I was trying to save you some time, but maybe haven't.